test/local/server.js

```diff
@@ -162,13 +178,15 @@ describe('lib/server', () => {
             return instance.inject({
               headers: {
                 'accept-language': 'fr-CH, fr;q=0.9, en-GB, en;q=0.5',
-                'user-agent': 'Firefox-Android-FxAccounts/34.0a1 (Nightly)'
+                'user-agent': 'Firefox-Android-FxAccounts/34.0a1 (Nightly)',
+                'x-forwarded-for': ' 194.12.187.0 , 194.12.187.0 '
               },
               method: 'POST',
               url: '/account/create',
               payload: {
                 features: [ 'signinCodes' ]
-              }
+              },
+              remoteAddress: '194.12.187.0'
```

Comment:
How likely is it for the ip address to be released and generate a new location? I believe `63.245.221.32` is an address that is owned by Mozilla and in Mountain View. Is there another ip we could potentially use and not Mountain View based? 

cc @jbuck

Reply:
I chose it for exactly this reason, because it's unlikely to change. The whole `194.12.187` block is owned by CERN. I figure they're not going anywhere.

Reply:
> The whole 194.12.187 block is owned by CERN. I figure they're not going anywhere.

Hopefully they don't get swallowed by a self-created black hole from the LHC.